enterprise/server/scheduling/scheduler_server/scheduler_server.go

```diff
@@ -479,6 +479,7 @@ func (h *executorHandle) startTaskReservationStreamer() {
 
 type executionNode struct {
 	executorID            string
+	executorHostID        string
```

Comment:
nit: Can you add a comment explaining the difference with these IDs

Reply:
Done